app/scripts/directives/fromFile.js

```diff
@@ -249,7 +255,7 @@ angular.module("openshiftConsole")
           var path;
           if ($scope.resourceKind === "Template" && $scope.templateOptions.process && !$scope.errorOccured) {
             var namespace = ($scope.templateOptions.add || $scope.updateResources.length > 0) ? $scope.projectName : "";
-            path = Navigate.createFromTemplateURL(resource, $scope.projectName, {namespace: namespace});
+            path = Navigate.createFromTemplateURL($scope.resourceList[0], $scope.projectName, {namespace: namespace});
```

Comment:
is this change necessary or did this get messed up in the merge conflict / rebase.  robb had just fixed an issue related to this call